streams/ae.m3u

```diff
@@ -63,8 +63,6 @@ https://dmisxthvll.cdn.mgmlcdn.com/dubaitvht/smil:dubaitv.stream.smil/playlist.m
 https://weyyak-live.akamaized.net/weyyak_dubai_tv/index.m3u8
 #EXTINF:-1 tvg-id="DubaiZaman.ae",Dubai Zaman (1080p) [Not 24/7]
 https://dmiffthvll.cdn.mangomolo.com/dubaizaman/smil:dubaizaman.stream.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="HawasTV.ae",Hawas TV (480p)
-https://jmc-live.ercdn.net/hawastvhd/hawastvhd.m3u8
```

Comment:
works

Reply:
Watch it for more than a minute and it will stop. The footage hasn't updated in a long time.

Reply:
ah okay.